Prace_domowe/Praca_domowa2/Mateusz_Ziemła/pd2.ipynb

```diff
@@ -0,0 +1,737 @@
+{
```

Comment:
<p>ale skoro te kategorie mają taki sam wpływ to może, że są zamienione na te same liczby? To tak jakby zrobić grupowanie</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>